contrib/terra-operator/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG base_image=classic-terra/core
```

Comment:
Remember to align the terradnode CRDs controller logic to take any ENV name changes into account. Otherwise we will spent hours chasing ghosts in k8s :P